pkg/registry/generic/etcd/etcd.go

```diff
@@ -271,10 +272,21 @@ func (e *Etcd) Update(ctx api.Context, obj runtime.Object) (runtime.Object, bool
 	if err != nil {
 		return nil, false, err
 	}
+	// If the object specified by the user does not have a resource version,
```

Comment:
Sorry, I meant specifically in `generic.Etcd` as an implementor you have to enable this function.  Prevents it being enabled by accident on new resources - I'd rather this be a deliberate decision.